.github/ISSUE_TEMPLATE/source_bug.yml

```
@@ -0,0 +1,101 @@
+name: 🐞 Issue/Bug report
+description: Report a source issue/bug in Tachiyomi
+title: "Source Name: short description"
+labels: [bug]
+body:
+
+  - type: checkboxes
+    id: acknowledgements
+    attributes:
+      label: Acknowledgements
+      description: Your issue will be closed if you haven't done these steps.
+      options:
+        - label: I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
+          required: true
+        - label: I have written a short but informative title.
+          required: true
+        - label: I have updated the app to version **[0.11.1](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
+          required: true
+        - label: I have updated all installed extensions.
+          required: true
+        - label: I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting-problems/).
+          required: true
+        - label: If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
+          required: true
+        - label: I will fill out all of the requested information in this form.
+          required: true
+
+  - type: input
+    id: tachiyomi-version
+    attributes:
+      label: Tachiyomi version
+      description: |
+        You can find your Tachiyomi version in **More → About**.
+      placeholder: |
+        Example: "0.11.1"
+    validations:
+      required: true
+
+  - type: input
+    id: android-version
+    attributes:
+      label: Android version
+      description: |
+        You can find this somewhere in your Android settings.
+      placeholder: |
+        Example: "Android 11"
+    validations:
+      required: true
+
+  - type: input
+    id: source
+    attributes:
+      label: Source information
+      description: |
+        List your source and extension version.
+
+        You can find your Extension version in **Browse → Extension**.
+      placeholder: |
+        Example: "Manga Plus 1.2.20"
+    validations:
+      required: true
+
+  - type: textarea
+    id: reproduce-steps
+    attributes:
+      label: Steps to reproduce
+      description: Provide an example of how to trigger the bug.
+      placeholder: |
+        Example:
+        1. First step
+        2. Second step
+        3. Bug occurs
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected behavior
+      placeholder: |
+        Example:
+        This should happen.
+    validations:
+      required: true
+
+  - type: textarea
+    id: actual-behavior
+    attributes:
+      label: Actual behavior
+      placeholder: |
+        Example: 
+        This happened instead.
```

Makes it inline with the other example blocks I added.
```suggestion
          "This happened instead..."
```